frame/support/src/storage/bounded_vec.rs

```diff
@@ -119,7 +119,17 @@ impl<T, S> BoundedVec<T, S> {
 		self.0
 	}
 
-	/// Exactly the same semantics as [`Vec::remove`].
+	/// Exactly the same semantics as `Vec::sort_by`.
```

Comment:
```suggestion
	/// Exactly the same semantics as [`slice::sort_by`].
```

Then rustdoc is happy :P